internal/backend/remote-state/s3/client.go

```diff
@@ -48,6 +53,8 @@ type RemoteClient struct {
 	kmsKeyID              string
 	ddbTable              string
 	skipS3Checksum        bool
+	lockFilePath          string
```

Comment:
Since `lockFilePath` is initialized to be `path`+`lockFileSuffix`, it doesn't need to be a field on the struct